### ISSUE TITLE
GE SIGNA DOI and Ring diameter incorrect 

### ISSUE THREAD

**Issue by pwadhwa351** [UCL/STIR#583]
GE SIGNA Depth of Interaction and Ring diameter values are incorrect in the GEHDF5Wrapper.cxx: initialisation of scanner parameters. The DOI value in the Scanner.cxx also needs to be changed.

**Comment by KrisThielemans** [comment]
Please do a PR on top of #564. I think we should put #564 on top of `release_4` once happy and merge that before 4.1. Without #564, GE Signa support will not be useful in 4.1 as it doesn't handle sinograms (sorry about that). @AnderBiguri is that ok for you?

**Comment by pwadhwa351** [comment]
Ok! Sounds fine!

**Comment by AnderBiguri** [comment]
@KrisThielemans sounds good for me. Should I just make sure we do all we know for RDF9 on that PR and open a new one for RDF 10?

**Comment by KrisThielemans** [comment]
@pw351 suggestion is for 
1. @AnderBiguri does steps 1-4 in https://github.com/UCL/STIR/pull/564#discussion_r440621400
2. you to modify `Scanner.cxx`, and handle step 5 discussed there by creating the scanner with `Scanner::get_scanner_from_name`, and then for whatever is in the header use the `Scanner::set` functions.

**Comment by pwadhwa351** [comment]
@KrisThielemans To understand better what is needed, I will modify initialise_scanner_from_HDF5 in the wrapper class? Within the function, I will be creating SIGNA using Scanner::get_scanner_from_name and then I set the remainder parameters available in the header using Scanner::set?

**Comment by KrisThielemans** [comment]
sounds correct to me.

**Comment by AnderBiguri** [comment]
@pwadhwa351 I might change this later, but please use the following to check for scanner.

```
if(check_current_signature(file)==GE_scanner::Signa)
{
   //is signa
}
else
{
  // it is some other scanner (but definitely a GE Hdf5)
}
```

**Comment by AnderBiguri** [comment]
@pwadhwa351 discard previous comment. As per kris's suggestion, now `check_current_signature` only returns a bool if its GE. You need to read the scanner name.

**Comment by KrisThielemans** [comment]
merged into #601
